src/components/cylc/GraphNode.vue

```diff
@@ -0,0 +1,157 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <g class="c-graph-node">
+    <!-- the task icon -->
+    <symbol :id="nodeID" viewBox="-40 -40 140 140">
+      <!--
+        Use a "symbol" for the task node in order to apply a viewBox to it.
+        This both contains it and makes it clickable.
+
+        NOTE: Due to the viewBox we use here the coordinate system ends up
+        offset by -20px. This doesn't impact most things, however, rotations
+        can be sensitive to this change causing the rotated elements to end up
+        in the wrong places. To counteract this we provide the coordinate offset
+        to the task component.
+      -->
+      <SVGTask
+        :task="task.node"
+        :modifierSize="0.5"
+        :startTime="startTime"
+        :coordinateOffset="-20"
+      />
+    </symbol>
+    <use
+      :href="`#${nodeID}`"
+      x="0" y="0"
+      width="150" height="150"
+      v-cylc-object="task.node"
+    />
+
+    <!-- the task name -->
+    <text
+      x="180" y="80"
+      font-size="50"
+    >
+      {{ task.name }}
+    </text>
+
+    <!-- the cycle point -->
+    <text
+      x="180" y="110"
+      font-size="25"
```

Comment:
I'm finding the cycle point text size to be very small
![image](https://user-images.githubusercontent.com/61982285/200885208-3efd7fda-8bd8-4031-8a10-b6e8ba656d19.png)

Reply:
Agreed, will boost along with the job squares.